### PR TITLE
pointing mobile devices to a proxy URL 

### PR DESCRIPTION
**Pull request by sanjay2916**
This PR is for Testing only and do not merge

### CODE REVIEW

File: src/commcare_cloud/ansible/roles/nginx/vars/cchq_ssl.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ nginx_sites:
    file_name: "{{ deploy_env }}_commcare"
    listen: "443 ssl http2{{ ' default_server' if deploy_env == primary_ssl_env else '' }}"
    root: "/var/www/html"
-   server_name: "{{ SITE_HOST }}"
+   server_name: "{{ SITE_HOST }} {{ ALTERNATE_HOSTS }}"
```
Comment (esoergel) [inline src/commcare_cloud/ansible/roles/nginx/vars/cchq_ssl.yml]: oh wow, this is sufficient for nginx to direct requests properly?  That's fantastic.

File: environments/staging/public.yml [hunk 2]
```diff
@@ -71,8 +71,10 @@ localsettings:
   ALLOWED_HOSTS:
     - j2mestaging.commcarehq.org
     - testserver
+    - sanjay1.commcarehq.org
   ANALYTICS_DEBUG: True
   ANALYTICS_LOG_LEVEL: "debug"
+  ALTERNATE_HOSTS: sanjay1.commcarehq.org
```
Comment (esoergel) [inline environments/staging/public.yml]: If this were a list of domains, would the commands below need to change?

**Comment by esoergel** [comment]
Change outline here: https://docs.google.com/document/d/1bpk4Fze6HmR1ryxq_ARUiNLEPElUpM0UNA4rdlLjveM/edit#

@sanjay2916

File: src/commcare_cloud/ansible/roles/nginx/vars/cchq_ssl.yml (hunk 1 above)
Comment (sanjay2916) [reply src/commcare_cloud/ansible/roles/nginx/vars/cchq_ssl.yml]: Yes Ethan

File: environments/staging/public.yml (hunk 2 above)
Comment (sanjay2916) [reply environments/staging/public.yml]: I am testing this and will update

Comment (sanjay2916) [reply environments/staging/public.yml]: we have to add list of domains with space separated by.

eg:  ALTERNATE_HOSTS: sanjay1.commcarehq.org sanjay2.commcarehq.org

Comment (esoergel) [reply environments/staging/public.yml]: I was thinking in particular of the certbot commands, which have lines like this:
```
-d {{SITE_HOST}} -d {{ NO_WWW_SITE_HOST }} -d {{ ALTERNATE_HOSTS }}
```
I imagine that'd render as
```
-d www.commcarehq.org -d commcarehq.org -d sanjay1.commcarehq.org sanjay2.commcarehq.org
```
When I suspect the command would require a `-d` before `sanjay2.commcarehq.org`

File: environments/icds-cas/public.yml [hunk 3]
```diff
@@ -3,8 +3,8 @@ daily_deploy_email: tech-announce-daily@dimagi.com
 
 elasticsearch_endpoint: '{{ groups.es0.0 }}'
 elasticsearch_cluster_name: 'icds-2.0'
-elasticsearch_version: 1.7.6
-elasticsearch_download_sha256: 78affc30353730ec245dad1f17de242a4ad12cf808eaa87dd878e1ca10ed77df.
+elasticsearch_version: 2.4.6
+elasticsearch_download_sha256: 5f7e4bb792917bb7ffc2a5f612dfec87416d54563f795d6a70637befef4cfc6f.
```
Comment (esoergel) [inline environments/icds-cas/public.yml]: What's with these changes?  Did you intend for them to be in a different PR?

Comment (sanjay2916) [reply environments/icds-cas/public.yml]: I will remove from this PR
